lib/core/network/api_service.dart

```diff
@@ -454,7 +454,16 @@ class APIService {
         body: jsonDecode(response.body) as Map<String, dynamic>,
       );
     }
-    return response.statusCode == 200;
+    final decodedBody = jsonDecode(response.body) as Map<String, dynamic>;
+    final isError = decodedBody['isError'] as bool;
+
+    if (response.statusCode == 200 && isError) {
```

Comment:
such combination (code 200 and isError true) looks really strange for me

Reply:
Its what we get from BE, technically the request is successful, accounts are created, but the isError field is true bc allowances are not successful

Reply:
yeah, I got the point) however it still looks like a hack from tech perspective :)